pages/calendar.vue

```diff
@@ -4,43 +4,86 @@
     justify-center
     align-center
   >
-    <categories-bar />
+    <categories-bar
+        @nextPressed="$refs.calendar.next()"
+        @prevPressed="$refs.calendar.prev()"
+     />
     <v-calendar
+        v-model="start"
         class="calendar"
+        :events="calendarEvents"
+        event-overlap-mode="stack"
+        :event-color="getEventColor"
+        event-text-color="black"
+        ref="calendar"
     />
   </v-layout>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
 import CategoriesBar from '@/components/CalendarComponents/categoriesBar.vue';
-import { allEvents } from '@/queries/Events';
+import { AllEvents } from '@/queries/Events';
 import Event from '@/models/Event.ts';
 
+interface CalendarEvent {
+    name: string; 
+    start: string;
```

Comment:
Why isn't a Date type?

Reply:
To format to the calendar time time stamp to display,
YYYY-MM-DD HH:MM

Reply:
Got it. You can do it with Date type aswell and for display use toString, but it's ok.